pkg/serverless/serverless.go

```diff
@@ -129,7 +127,7 @@ func WaitForNextInvocation(stopCh chan struct{}, daemon *daemon.Daemon, id regis
 	if payload.EventType == Invoke {
 		functionArn := removeQualifierFromArn(payload.InvokedFunctionArn)
 		callInvocationHandler(daemon, functionArn, payload.DeadlineMs, safetyBufferTimeout, payload.RequestID, handleInvocation)
-	} else if payload.EventType == Shutdown || payload.EventType == Failure {
+	} else if payload.EventType == Shutdown {
```

Comment:
Removed the other condition here since this was a mistake that I added in a previous PR, i.e., no actual Failure event exists from the extension API